server/src/main/scala/org/broadinstitute/dsp/workbench/welder/server/ObjectService.scala

```diff
@@ -98,17 +99,18 @@ class ObjectService(
     for {
       context <- storageLinksAlg.findStorageLink(req.localObjectPath)
       current <- timer.clock.realTime(TimeUnit.MILLISECONDS)
+      gsPath = getGsPath(req.localObjectPath, context)
+      hashedLockedByCurrentUser <- hashString(lockedByString(gsPath.bucketName, config.ownerEmail))
       metadata = Map(
-        GoogleStorageAlg.LAST_LOCKED_BY -> config.ownerEmail.value,
+        GoogleStorageAlg.LAST_LOCKED_BY -> hashedLockedByCurrentUser.asString,
```

Comment:
there're a few different places where `config.ownerEmail` is used. I'm thinking it's better to change `ownerEmail` to `HashedLockedBy` in config, and do the hashing in https://github.com/DataBiosphere/welder/blob/master/server/src/main/scala/org/broadinstitute/dsp/workbench/welder/server/Config.scala#L21

Reply:
i agree it would be nice to push that up to config.scala but the hashing is a combo of the gcs bucket name and the user's email. since we won't know which bucket they're syncing to until the lock request comes in (and there could be multiple buckets that welder is juggling), this wouldn't be possible